src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const successContainer = document.createElement('div');
+const errorContainer = document.createElement('div');
+
+successContainer.classList.add('message');
+successContainer.textContent = 'Promise was resolved!';
+
+errorContainer.classList.add('error-message');
+errorContainer.classList.add('message');
+errorContainer.textContent = 'Promise was rejected!';
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', () => {
+    resolve(document.body.append(successContainer));
```

Comment:
Instead of resolving with 'document.body.append(successContainer)', just resolve and move the 'document.body.append()' operation to the '.then()' block.